packages/access-api/test/provider-add.test.js

```diff
@@ -97,6 +99,100 @@ for (const accessApiVariant of /** @type {const} */ ([
       })
     })
   })
+
+  if (
+    ['provider/add', 'access/delegate'].every((f) =>
+      accessApiVariant.features.has(f)
+    )
+  ) {
+    it('provider/add allows for access/delegate', async () => {
+      const space = await principal.ed25519.generate()
+      const issuer = await accessApiVariant.issuer
+      const service = await accessApiVariant.audience
```

Comment:
when @gozala and I were debugging earlier, he asked a good question e.g. to make sure that e.g. "service isn't same as issuer" or something.

I don't think they are, but one thing to keep in mind is that `service` may have been created using `withDID`:

note: this can be created in
https://github.com/web3-storage/w3protocol/blob/main/packages/access-api/test/helpers/context.js#LL50C3-L52C4
like
```
  const servicePrincipal = Signer.parse(bindings.PRIVATE_KEY).withDID(
    bindings.DID
  )
```